financial-connections/res/values/strings.xml

```diff
@@ -105,6 +105,12 @@
     <string name="stripe_accountpicker_singleaccount_description">%1$s only needs one account at this time.</string>
     <string name="stripe_accountpicker_singleaccount_description_nobusinessname">This merchant only needs one account at this time.</string>
     <string name="stripe_account_picker_select_all_accounts">Select all accounts</string>
+    <string name="stripe_networking_signup_terms">Stripe’s Terms and Privacy Policy</string>
+    <string name="stripe_networking_signup_phone_number_disclaimer">We’ll use this to verify and manage your account.</string>
+    <string name="stripe_networking_signup_bullet1">Connect your account faster on %1$s and everywhere Link is accepted.</string>
+    <string name="stripe_networking_signup_bullet2">Link encrypts your data and never shares your login details.</string>
+    <string name="stripe_networking_signup_cta_save">Save to Link</string>
+    <string name="stripe_networking_signup_cta_negative">No thanks</string>
```

Comment:
I believe Stripe.js copy is "Not now"

Reply:
Good catch ❤️ https://github.com/stripe/stripe-android/pull/6137/commits/b2872f2c96a88aaf53137ff0d2d8abda047696f0